code/game/objects/effects/landmarks_events.dm

```diff
@@ -0,0 +1,129 @@
+/*
+Landmark to be spawned by GMs and admins when running events.
+Do NOT directly spawn "" or buildmode spawn this.
+Use the "Manage Event Triggers" verb under "Eventkit" category.
+Admin verb is called by code\modules\admin\verbs\event_triggers.dm
+*/
+/obj/effect/landmark/event_trigger
+	name = "ONLY FOR EVENTS. DO NOT USE WHEN MAPPING"
+	desc = "Please notify staff if you see this!"
+	var/creator_ckey = ""
+	var/isTeamwork = FALSE //Whether to notify creator or whole team.
+	var/isRepeating = FALSE
+	var/coordinates = ""
+	var/cooldown = 0 //in seconds
+	var/last_trigger = 0
+	var/isLoud = FALSE
+	var/isNarrate = FALSE
+
+/obj/effect/landmark/event_trigger/New()
+	..()
+	coordinates = "(X:[loc.x];Y:[loc.y];Z:[loc.z])"
+	invisibility = 100
```

Comment:
Why is invisibility set in new and not in the main definition itself, if its never touched anyway?

Reply:
The reason it's set in New() because the type it inherits from (landmark) sets it at 101, and I want it at 100 for ability to see it  with see invisibility

Reply:
Wait, landmark sets it in new? Why does THAT do that? Why not change that?